README.md

```diff
@@ -126,6 +126,15 @@ tags = get_tags(MyEstimator())
 
 Which uses `sklearn.utils.get_tags` under the hood from scikit-learn 1.6+.
 
+#### `check_estimator` and `parametrize_with_checks` functions
+
+The new tags don't include a `_xfail_checks` tags, and instead, the tests which are
+expected to fail are directly passed to the `check_estimator` and
+`parametrize_with_checks` functions. The two functions available in this package are
+compatible with the new signature, and patch the estimator in older scikit-learn
+versions to include the expected failed checks in their tags so that you don't need
+to include them both in your tests and in your `_xfail_checks` old tags.
```

Comment:
Just a small snippet would be worth. Do we want the `patched_more_tags` to be really public?

Reply:
There's no public or private thing here, and there's no deprecation cycle. We're always compatible with the latest sklearn, and we'd never be raising a deprecation warning here. From version to version, things will break.

Reply:
I was thinking more about exposing a function that do exist in scikit-learn.

Reply:
I'm not sure what you want me to do here

Reply:
Oh `_patched_more_tags` starts with an underscore. I probably misread.

Reply:
No you didn't, I added the underscore (albeit grumpily 😅) after your review!

Reply:
I don't want to make you grumpy a Sunday :)